Servers/UI/OJS.Servers.Ui/ClientApp/src/components/contests/contests-filters/ContestFilters.tsx

```diff
@@ -28,25 +27,19 @@ interface IFiltersGroup {
 const ContestFilters = ({ onFilterClick }: IContestFiltersProps) => {
     const maxFiltersToDisplayCount = 3;
     const { search } = useLocation();
-    const { clearPageValue } = usePages();
+    const { actions: { clearParams } } = useUrlParams();
     const [ selectValue, setSelectValue ] = useState('');
     const [ filtersGroups, setFiltersGroups ] = useState<IFiltersGroup[]>([]);
     const [ defaultSelected, setDefaultSelected ] = useState('');
     const [ filteredStrategyFilters, setFilteredStrategyFilters ] = useState<IFilter[]>([]);
-    const [ searchParams, setSearchParams ] = useSearchParams();
+    const [ searchParams ] = useSearchParams();
     const [ isLoaded, setIsLoaded ] = useState(false);
 
     const {
         state: { possibleFilters },
-        actions: {
-            toggleParam,
-            clearFilters,
-            clearSorts,
-        },
+        actions: { toggleParam },
     } = useContests();
 
-    const { actions: { clearBreadcrumb } } = useCategoriesBreadcrumbs();
-
     React.useEffect(() => {
```

Comment:
Remove "React" from React.useEffect. We have already defined it and there is no need for it.